### PR TITLE
add coxreg support and AIC for models from eha package

### PR DESCRIPTION
**Pull request by sdaza**

### CODE REVIEW

File: R/extract.R [hunk 1]
```diff
@@ -4434,19 +4434,19 @@ extract.vglm <- function(model, include.loglik = TRUE, include.df = TRUE,
...
 }
 
-setMethod("extract", signature = className("vglm", "VGAM"), 
+setMethod("extract", signature = className("vglm", "VGAM"),
     definition = extract.vglm)
 
 
 # extension for weibreg objects (eha package)
-extract.weibreg <- function(model, include.loglik = TRUE, include.lr = TRUE, 
+extract.weibreg <- function(model, include.aic = TRUE, include.loglik = TRUE, include.lr = TRUE,
```
Comment (leifeld) [inline R/extract.R]: make sure this is within a width of 80 characters -- here and in all other places you edited

**Comment by sdaza** [comment]
It's done. It's working properly in my laptop.

**Comment by leifeld** [comment]
Great! Can you increment the number of supported models in the caption of the table in the vignette from 75 to 76? I think there are two instances in that caption.

**Comment by leifeld** [comment]
Perfect, thanks!

**Pull request merged**